docs/chainlink-nodes/configuration-variables.md

```diff
@@ -1594,80 +1600,100 @@ Includes gas price in calls to `checkUpkeep()` when set to `true`.
 
 ### KEEPER_GAS_PRICE_BUFFER_PERCENT
 
+> 🚧 **ADVANCED**
+>
+> Do not change this setting unless you know what you are doing.
+
 - Default: `"20"`
 
 `KEEPER_GAS_PRICE_BUFFER_PERCENT` adds the specified percentage to the gas price used for checking whether to perform an upkeep. Only applies in legacy mode (EIP-1559 off).
 
 ### KEEPER_GAS_TIP_CAP_BUFFER_PERCENT
 
+> 🚧 **ADVANCED**
+>
+> Do not change this setting unless you know what you are doing.
+
 - Default: `"20"`
 
 `KEEPER_GAS_TIP_CAP_BUFFER_PERCENT` adds the specified percentage to the gas price used for checking whether to perform an upkeep. Only applies in EIP-1559 mode.
 
 ### KEEPER_BASE_FEE_BUFFER_PERCENT
 
+> 🚧 **ADVANCED**
+>
+> Do not change this setting unless you know what you are doing.
+
 - Default: `"20"`
 
 Adds the specified percentage to the base fee used for checking whether to perform an upkeep. Applies only in EIP-1559 mode.
 
 ### KEEPER_MAXIMUM_GRACE_PERIOD
 
-**ADVANCED**
-
-Do not change this setting unless you know what you are doing.
+> 🚧 **ADVANCED**
+>
+> Do not change this setting unless you know what you are doing.
 
 - Default: `"100"`
 
 The maximum number of blocks that a keeper will wait after performing an upkeep before it resumes checking that upkeep
 
 ### KEEPER_REGISTRY_CHECK_GAS_OVERHEAD
 
-**ADVANCED**
-
-Do not change this setting unless you know what you are doing.
+> 🚧 **ADVANCED**
+>
+> Do not change this setting unless you know what you are doing.
 
 - Default: `"200000"`
 
 The amount of extra gas to provide checkUpkeep() calls to account for the gas consumed by the keeper registry.
 
 ### KEEPER_REGISTRY_PERFORM_GAS_OVERHEAD
 
-**ADVANCED**
-
-Do not change this setting unless you know what you are doing.
+> 🚧 **ADVANCED**
+>
+> Do not change this setting unless you know what you are doing.
 
 - Default: `"150000"`
```

Comment:
Please change this to 300000, we recently changed the default.

Reply:
Done.